resharper/resharper-unity/src/UnityImplicitUsageHighlightingContributor.cs

```diff
@@ -0,0 +1,187 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Drawing;
+using JetBrains.Application.Settings;
+using JetBrains.Application.UI.Controls.BulbMenu.Anchors;
+using JetBrains.Application.UI.Controls.BulbMenu.Items;
+using JetBrains.DocumentModel;
+using JetBrains.ProjectModel;
+using JetBrains.ProjectModel.DataContext;
+using JetBrains.ReSharper.Daemon.CodeInsights;
+using JetBrains.ReSharper.Feature.Services.Bulbs;
+using JetBrains.ReSharper.Feature.Services.Daemon;
+using JetBrains.ReSharper.Feature.Services.Intentions;
+using JetBrains.ReSharper.Feature.Services.Resources;
+using JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Errors;
+using JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Stages.Highlightings;
+using JetBrains.ReSharper.Plugins.Unity.CSharp.Feature.Services.Bulbs;
+using JetBrains.ReSharper.Plugins.Unity.CSharp.Feature.Services.QuickFixes;
+using JetBrains.ReSharper.Plugins.Unity.Settings;
+using JetBrains.ReSharper.Psi;
+using JetBrains.ReSharper.Psi.CSharp.Tree;
+using JetBrains.ReSharper.Psi.Tree;
+using JetBrains.ReSharper.Resources.Resources.Icons;
+using JetBrains.TextControl;
+using JetBrains.TextControl.DocumentMarkup;
+using JetBrains.Util;
+
+[assembly: RegisterHighlighter(UnityHighlightingAttributeIds.UNITY_IMPLICIT_USAGE_BOLD_ATTRIBUTE, GroupId = UnityHighlightingGroupIds.Unity,
+    EffectType = EffectType.TEXT, FontStyle = FontStyle.Bold, Layer = HighlighterLayer.SYNTAX + 1)]
+
+namespace JetBrains.ReSharper.Plugins.Unity
+{
+    [StaticSeverityHighlighting(Severity.INFO, "UnityGutterMarks", Languages = "CSHARP", OverlapResolve = OverlapResolveKind.NONE)]
+    public class UnityImplicitBoldHighlighting : ICustomAttributeIdHighlighting
```

Comment:
Take the "Bold" out of the name. Use something like `UnityImplicitlyUsedIdentifierHighlighting`.

Reply:
Move to its own file, too: `CSharp\Daemon\Stages\Highlightings`